src/Bzip.php

```diff
@@ -54,7 +54,14 @@ public function extract($pathExtracted)
 
         $output = fopen($pathExtracted, 'w');
         while (!feof($file)) {
-            fwrite($output, fread($file, 1024 * 1024));
+            $content = fread($file, 1024 * 1024);
+
+            if (false === $content) {
+                $this->error = "fread failed";
+                return false;
```

Comment:
I expect this to solve the problem.

Only question would be: should we do any explicit cleanup? E.g. delete the "most likely broken" output file? And maybe close all file handles, even though the should (afaik) also be closed by the garbage collection once the function returns?